### ISSUE TITLE
Webp images not been created on images that exist prior to webp command path being defined correctly

### ISSUE THREAD

**Issue by davecoggins** [nystudio107/craft3-imageoptimize#98]
Hi

To recreate this issue I had the webp bin path set to the default which was not correct for my local setup. I added the config file to my config directory and in an array for my local environment set the path correctly. Checking the settings page I have all green ticks for everything apart from mozjpeg.

I cleared the cache and regenerated the images but no webp images were created. However if I upload a new image the webp version is created succesfully.

So to summarize it looks like webp images are not being created for images that already exist in the asset volume.

Awesome plugin btw :)

Cheers
Dave

**Comment by khalwat** [comment]
I think this is because of how Craft CMS Asset Transforms work (which Image Optimize is layered on top of). If the transform already exists, it won't bother recreating it... so then Image Optimize never gets called, so the webp variant isn't created.

What I'd suggest doing is going to Utilities -> Clear Caches and tick all of the Asset related caches, click on **Clear Caches** and then to force it to resave all of the images, go to Settings -> Asset Volumes, click on the Asset Volume where you added the Optimized Images field, and click Save.

This will kick off a resave job to recreate all of the images.
